src/ripple/app/tx/impl/LedgerStateFix.cpp

```diff
@@ -0,0 +1,98 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2024 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/app/tx/impl/LedgerStateFix.h>
+
+#include <ripple/app/tx/impl/details/NFTokenUtils.h>
+#include <ripple/ledger/View.h>
+#include <ripple/protocol/Feature.h>
+#include <ripple/protocol/Indexes.h>
+#include <ripple/protocol/Protocol.h>
+#include <ripple/protocol/TxFlags.h>
+
+namespace ripple {
+
+NotTEC
+LedgerStateFix::preflight(PreflightContext const& ctx)
+{
+    if (!ctx.rules.enabled(fixNFTokenPageLinks))
+        return temDISABLED;
+
+    if (ctx.tx.getFlags() & tfUniversalMask)
+        return temINVALID_FLAG;
+
+    if (auto const ret = preflight1(ctx); !isTesSuccess(ret))
+        return ret;
+
+    switch (ctx.tx[sfLedgerFixType])
+    {
+        case FixType::nfTokenPageLink:
+            if (!ctx.tx.isFieldPresent(sfOwner))
+                return temINVALID;
+            break;
+
+        default:
+            return tefINVALID_LEDGER_FIX_TYPE;
+    }
+
+    return preflight2(ctx);
+}
+
+XRPAmount
+LedgerStateFix::calculateBaseFee(ReadView const& view, STTx const& tx)
+{
+    // The fee required for AccountDelete is one owner reserve.
```

Comment:
why does it say AccountDelete?

Reply:
Good eyes!  That's a copy-and-paste-o and needs to be fixed.